kfai_sql_chemistry/query/constructor.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+import logging
+
+from sqlalchemy.orm import Session, sessionmaker
+
+from kfai_sql_chemistry.db.main import engines
+
+logger = logging.getLogger(__name__)
+
+
+class QueryConstructor:
+    def __init__(self, column: str, table_name: str, value:str):
```

Comment:
i think we'll have to iterate on the API a bit here. SqlAlchemy Models are preferred over string values/references to create queries.